package.json

```diff
@@ -155,7 +84,7 @@
     "snoop": "1.0.4",
     "source-map": "0.7.4",
     "svgo": "3.3.2",
-    "syncpack": "12.3.3",
+    "syncpack": "14.0.0-alpha.19",
```

Comment:
should we really be depending on an alpha version here? Is there something broken that needs this version to fix?